src/main/java/com/shippingcostestimator/enterprise/PackageEstimatorController.java

```diff
@@ -39,7 +39,7 @@ public String index() {
     /***
      * Handle the saveEstimate ("/saveEstimate") endpoint and save a Shipment object using data specified by the user.
      * @param shipment
-     * @return
+     * @return start.html page.
      */
```

Comment:
Added return description for JavaDoc. [Ref](https://medium.com/@rhamedy/a-short-summary-of-java-coding-best-practices-31283d0167d3)